### ISSUE TITLE
Give option for setting tmp dir

### ISSUE THREAD

**Issue by tomvantilburg** [sharedstreets/sharedstreets-builder#15]
Currently fink uses the systems tmp dir for storing intermediate files. 
It would be nice when the user can give a custom tmp dir for this so it would be possible to dedicate larger disks for the process.

I know there is an option in flink `taskmanager.tmp.dirs` but it is not clear how to set this.

**Comment by kpwebb** [comment]
@tomvantilburg yes -- we've been configuring this in our planet.pbf builds as it takes about 4TB of scratch disk space to process a global tile set. 

That said, I'd love to know more about you're using the builder -- we've made a global z12 tile set that's hosted on S3 (we're in the process of moving it to the AWS public data sets program). I'm curious if that address your need or if you're doing something different with the app?

**Comment by tomvantilburg** [comment]
Does the tileset involve simplification/aggregation? Or is it just about the bounds of every tile?
Would definitely be interested in a service, but for the rest just playing around.
